ReText/window.py

```diff
@@ -462,6 +462,9 @@ def tabFileNameChanged(self, tab):
 		if tab == self.currentTab:
 			if tab.fileName:
 				self.setWindowTitle("")
+				if globalSettings.titleBarFullPath:
+					fullpath = QFileInfo(self.currentTab.fileName).filePath()
```

Comment:
`self.currentTab.fileName` should be already a canonical file path, it is set here:

https://github.com/retext-project/retext/blob/748a1041a48f4b5c95f6b3f586a761cfa32721e9/ReText/window.py#L821